ndindex/ellipsis.py

```diff
@@ -79,3 +79,6 @@ def newshape(self, shape):
             shape = (shape,)
 
         return shape
+
+    def as_subindex(self, index):
```

Comment:
These public methods should have docstrings everywhere in this PR

Reply:
This is documented on the superclass.

Reply:
Do docstrings really inherit like that?

Reply:
Yeah, `help()` will show the superclass docstring if the subclass doesn't have one, and Sphinx autodoc also inherits. For example, search for "newshape" [here](https://quansight.github.io/ndindex/api.html). I only documented it once on the superclass because otherwise they would all be mostly redundant anyway.

Reply:
Oh cool. I didn't know that